extensionsGallery-insider.json

```diff
@@ -4259,6 +4259,63 @@
 					],
 					"statistics": [],
 					"flags": "preview"
+				},
+				{
+					"extensionId": "84",
+					"extensionName": "azcli",
+					"displayName": "Azure CLI",
+					"shortDescription": "Provides Azure CLI capabilities for use in Azure Data Studio",
+					"publisher": {
+						"displayName": "Microsoft",
+						"publisherId": "Microsoft",
+						"publisherName": "Microsoft"
+					},
+					"versions": [
+						{
+							"version": "0.1.0",
+							"lastUpdated": "9/3/2021",
```

Comment:
Should this be 8/3/2021 (aug rather than sept)?

Reply:
Hah, whoops. I'm just ready for it to be fall already apparently